modules/node-app-service/main.tf

```diff
@@ -70,6 +70,32 @@ resource "azurerm_linux_web_app" "web_app" {
   }
 
   virtual_network_subnet_id = var.outbound_vnet_connectivity ? var.integration_subnet_id : null
+
+  # auth settings
+  auth_settings_v2 {
+    auth_enabled             = true
```

Comment:
This should be configurable - auth_config should be optional, and this only true if auth_config is provided.

Reply:
Updated the value as bool variable